### DISCUSSION TITLE
Middleware | Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#17] [Announcements]
# Middleware | Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/middleware/

**Comment by I-SMAF** [comment]
Почему то, что вы назвали "функция промежуточной обработки" реализует итоговую реализацию (реакцию)?

 где промежуточность?

как мутировать request (что ещё за "req"...) для всех обработчиков разом в абстракции от эндпоинта?
